src/platform/graphics/shader-processor.js

```diff
@@ -353,24 +354,64 @@ class ShaderProcessor {
         return block;
     }
 
-    static processAttributes(attributeLines, shaderDefinitionAttributes) {
+    // extract count from type ('vec3' => 3, 'float' => 1)
+    static getTypeCount(type) {
```

Comment:
not crazy about this function name, though probably not very important

Reply:
yeah, but its internal private, and I didnt want a very long name :)